libenkf/src/enkf_config_node.c

```diff
@@ -603,6 +607,35 @@ const char * enkf_config_node_get_enkf_infile( const enkf_config_node_type * con
   return path_fmt_get_fmt( config_node->enkf_infile_fmt );
 }
 
+
+const char * enkf_config_node_get_init_file(enkf_config_node_type * config_node, const path_fmt_type * runpath_fmt) {
```

Comment:
Disclaimer: I used quite long time to disentange / recap this myself. 

The name `init_file` - which was also used in the previous code - is unfortunate, because it hints at initialization, which is an important task which applies to all the ert node types. However - this is a very specialized use case; only applicable when exporting a field - e.g. `PORO` or `PERMX`. Ert ony internalizes the active cells of a field, by using the filename given here the export routine gets access to a "templatee/fill in" field which can be used for the inactive cells, the same field is used for all realizations. 

Suggestions:

1. This function is renamed to something like `enkf_config_node_get_FIELD_fill_file()`
2. The `getFieldInitFile()` is completely removed from the `EnkfMain` class - instead the two call sites: [here](https://github.com/Ensembles/ert/blob/master/python/python/ert_gui/shell/export.py#L59) and [here](https://github.com/Ensembles/ert/blob/master/python/python/ert_gui/tools/export/export_model.py#L47) use the config node implementation directly.